resources/translations/eng.ini

```diff
@@ -98,6 +98,14 @@ commands.unban.usage=/pardon <name>
 commands.unbanip.invalid=You have entered an invalid IP address
 commands.unbanip.success=Unbanned IP address {%0}
 commands.unbanip.usage=/pardon-ip <address>
+commands.weather.clear=Changing to clear weather
+commands.weather.disabled=Weather Cycle isn't enabled in this world.
+commands.weather.query=Weather state is: %s
```

Comment:
This is not the correct parameter format for PM, you can't just copy paste directly from vanilla language files